raiden/tests/utils/mocks.py

```diff
@@ -35,8 +36,8 @@ def __init__(self):
         # let's make a single mock token network for testing
         self.token_network = MockTokenNetwork()
 
-    def payment_channel(self, token_network_address, channel_id):
-        return MockPaymentChannel(self.token_network, channel_id)
+    def payment_channel(self, canonical_identifier: CanonicalIdentifier):
+        return MockPaymentChannel(self.token_network, canonical_identifier.channel_identifier)
```

Comment:
Shouldn't you use `canonical_identifier.token_network` here instead of `self.token_network`?

Reply:
the `MockPaymentChannel` sets `self.token_network` (not a `TokenNetworkAddress`) in `__init__`, so I think, in scope of the current mocking, this is okay.